Modules/syslogmodule.c

```diff
@@ -195,6 +214,10 @@ syslog_syslog_impl(PyObject *module, int group_left_1, int priority,
 
     /*  if log is not opened, open it now  */
     if (!S_log_open) {
+        if (!is_main_interpreter()) {
+            PyErr_SetString(PyExc_RuntimeError, "unable to use syslog.syslog at non-main interpreter for the first time.");
```

Comment:
As an user, I don't understand this error message. As an user, what should I do to fix the error?

In Python, usually error messages don't end with a trailing dot.

Reply:
> As an user, I don't understand this error message. As an user, what should I do to fix the error?

I revised the error message, Can you please take a look? Would you like to suggest better messages?